pom.xml

```diff
@@ -54,11 +56,29 @@
 	</distributionManagement>
 
 	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-batch</artifactId>
+		</dependency>
+  		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-security</artifactId>
+		</dependency>
```

Comment:
It's not something that needs to be solved now, but it would be nice if this didn't depend on Spring Security. We would need to find some way for the application to make route and method security decisions though.